src/engraving/dom/engravingitem.cpp

```diff
@@ -1451,7 +1451,10 @@ PropertyPropagation EngravingItem::propertyPropagation(const EngravingItem* dest
     const Staff* destinationStaff = destinationItem->staff();
 
     if (sourceScore == destinationScore) {
-        if (sourceStaff != destinationStaff && (propertyId == Pid::VISIBLE || propertyGroup(propertyId) == PropertyGroup::POSITION)) {
+        const bool sameStaff = sourceStaff != destinationStaff;
```

Comment:
I think you meant the opposite?

Reply:
Ah yes - the mistake is in the variable name.